docs/_includes/common/carousel.html

```diff
@@ -81,16 +81,18 @@ <h4 class="clipped" aria-live="polite">
                             <use xlink:href="#icon-carousel-prev"></use>
                         </svg>
                     </button>
-                    <ul class="carousel__list">
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 16px);">Card 1</li>
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 16px);">Card 2</li>
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 16px);">Card 3</li>
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 0px);">Card 4</li>
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 16px);">Card 5</li>
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 16px);">Card 6</li>
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 16px);">Card 7</li>
-                        <li class="demo-filmstrip__card" style="width: calc(25% - 0px);">Card 8</li>
-                    </ul>
+                    <div class="carousel__fade-right">
```

Comment:
The example code needs updating too.

Reply:
Can you give an example? I am not sure what you mean by example code. I updated both tests page and the jekyll pages. I only updated a couple of examples that might be using this fade and not all. 
Is there a storybook thing I'm missing? 
The other pages in ds4/ds6 just reference this `common/carousel.html` page.

Reply:
Sorry, I mean the example code underneath:

<img width="1114" alt="Screen Shot 2020-01-30 at 9 38 39 AM" src="https://user-images.githubusercontent.com/38065/73474973-57e9d200-4344-11ea-9ac7-e42508e9f76d.png">